docs/data/data-grid/editing-legacy/editing.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Data Grid - Editing (legacy)
+---
+
+# Data Grid - Editing (legacy)
+
+<p class="description">The data grid has built-in edit capabilities that you can customize to your needs.</p>
+
+## Cell editing
+
+Cell editing allows editing the value of one cell at a time.
+Set the `editable` property in the `GridColDef` object to `true` to allow editing cells of this column.
```

Comment:
Along with swapping out "allow," I would rearrange this sentence to place the most important clause in front:

```suggestion
To enable cell editing within a column, set the `editable` property in the `GridColDef` object to `true`.
```